fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -356,6 +358,8 @@ public Response createOrReplaceObjectRdf(
                 }
                 throw new ClientErrorException("Invalid Content Type " + requestContentType, UNSUPPORTED_MEDIA_TYPE);
             }
+        } catch (final Exception e) {
```

Comment:
It seems like we only care about `IOExceptions` when looking for storage issues. Is there a reason we are catching all `Exception`s here?